### DISCUSSION TITLE
MJPEG does not work

### DISCUSSION THREAD

**Discussion by KaiBuesing** [jtee3d/prusa_connect_rtsp#1] [Q&A] [answered]
Hi Joseph,

everything works perfectly with RTSP. I tried using MJPEG instead of RTSP today. Unfortunately that does not work :-( 

That is a look to the Logs : 

<img width="1009" alt="image" src="https://github.com/jtee3d/prusa_connect_rtsp/assets/113059963/886636b6-a13f-48a1-84b0-9f812ae15b07">

And here my Docker Compose : 

version: '3.8'
services:
  prusa_connect_rtsp:
    image: jtee3d/prusa_connect_rtsp:latest
    restart: always
    environment:
      CAMERA_URLS: >
        http://192.168.200.81:81/cam.mjpeg,
        http://192.168.200.82:81/cam.mjpeg,
        http://192.168.200.83:81/cam.mjpeg,
        http://192.168.200.84:81/cam.mjpeg
      TOKENS: >
        iMVMHqWXUFdrxkutp5K6,
        TL6DdrBA5ZNs8UtLr6KN,
        BZDo5N5PbMcOtiNmD1aY,
        LJhu5YXyM2v3g5xNWcEN

Any idea ?

But all in all a great job!

**Comment by BillyAB** [comment, marked as answer]
Hi @KaiBuesing,

It looks like you need to re-pull the latest docker image. `docker pull jtee3d/prusa_connect_rtsp`

The new version of the script in the latest commit and the latest docker image has slightly different wording under the ascii logo, the screenshot you have provided is the old text, which suggests you are using an old image.

Give it a re-pull and try again, should get it working for you.

**Reply by jtee3d** [reply]
The text beneath the ascii logo should be `This script sends snapshots of RTSP and MJPEG streams to Prusa Connect.`. In the next release, I'll include the version number in the console output.

**Reply by KaiBuesing** [reply]
The tip was the solution. It works now, thank you very much!

<img width="774" alt="image" src="https://github.com/jtee3d/prusa_connect_rtsp/assets/113059963/620a4ba6-6d10-4b04-86a2-889870d40d93">
